test/main.test.js

```diff
@@ -490,6 +494,28 @@ describe('bundling single js feed', () => {
             .send(['completelyfake.json'])
             .expect(409));
 
+    test('invalid json response should error correctly', async () => {
+        // Missing closing `}`
+        router.sink.set('invalid.json', Buffer.from('{ "foo": "bar"'));
+
+        const eventPromise = new Promise(resolve => {
+            router.once('request error', async (_, __, ___, ____, error) =>
+                resolve(error)
+            );
+        });
+
+        const [errorEvent] = await Promise.all([
+            eventPromise,
+            supertest(server)
+                .post('/bundle/js')
+                .send(['invalid.json'])
+                .expect(500),
```

Comment:
because of error handling in express, it was still 500 to the user with the old code.

I found no really clean way for asserting that our last middleware (which sends the response) did not crash, so I assert that the emitted error from the middleware before that middleware is a `Boom`